addons/im_livechat/models/discuss_channel_member_history.py

```diff
@@ -0,0 +1,34 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class DiscussChannelMemberHistory(models.Model):
+    _name = "discuss.channel.member.history"
```

Comment:
If we only define/use it in livechat, we might as well name it `im_livechat.member.history`.
Or `im_livechat.discuss.channel.member.history`, or Or `im_livechat.session.member.history` or something like that.